std/socket.d

```diff
@@ -2923,7 +2923,7 @@ public:
     }
 
 
-    private static void _close(socket_t sock) @system nothrow @nogc
+    private static void _close(scope socket_t sock) @system nothrow @nogc
```

Comment:
`socket_t` is an `int`, no need to add `scope` here

Reply:
done